k8s.io/test.py

```diff
@@ -66,6 +66,11 @@ def test_go(self):
 
     # TODO: external go-get ???
 
+    def test_apt_test(self):
+        # FIXME: https://apt.kubernetes.io is not on the cert
+        base = 'http://apt.kubernetes.io'
+        self.assert_redirect(base, 'http://storage.googleapis.com/k8s-releases/apt/')
```

Comment:
maybe do `self.assert_redirect(base + '/$id', 'http://storage.googleapis.com/k8s-releases/apt/$id', id=rand_num())` to test that paths are redirecting properly.